evap/staff/views.py

```diff
@@ -1205,6 +1212,18 @@ def notify_reward_points(grantings, **_kwargs):
 
         return redirect("staff:semester_view", semester.id)
 
+    contributor_questionnaire_pairs = [
+        (answer.contribution.contributor, answer.question.questionnaire)
+        for answer_cls in [TextAnswer, RatingAnswerCounter]
```

Comment:
Might be worth adding the `set(Answer.__subclasses__()) == {TextAnswer, RatingAnswerCounter}` assertion, too.